src/conn/conn_prefetch.c

```diff
@@ -186,6 +186,11 @@ __wt_conn_prefetch_queue_push(WT_SESSION_IMPL *session, WT_REF *ref)
         WT_ERR(EBUSY);
     }
 
+    /*
+     * On top of indicating the leaf page is now in the prefetch queue, the prefetch flag also
+     * guarantees the corresponding internal page cannot be evicted until prefetch has processed the
+     * leag page. This flag is checked when eviction reviews an internal page for active children.
```

Comment:
`leag` to `leaf`.

Reply:
Looks like I reverted 2a9c6dd86d7a4e5eca560a4d9f93a38c56c84d87 by mistake